File: home/src/main/java/com/android/home/HomeFragment.java
package com.android.home;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.android.base.BaseFragment;

/**
 * created by jiangshide on 2019-07-24.
 * email:18311271399@163.com
 */
public class HomeFragment extends BaseFragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return setView(R.layout.fragment_home, true);
    }
}

File: business/zd112/home/src/main/java/com/android/home/HomeFragment.kt
package com.android.home

import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import com.android.base.BaseFragment
import com.android.blog.BlogFragment

/**
 * created by jiangshide on 2019-07-24.
 * email:18311271399@163.com
 */
class HomeFragment : BaseFragment() {

  override fun onViewCreated(
    view: View,
    savedInstanceState: Bundle?
  ) {
    super.onViewCreated(view, savedInstanceState)
    createTabLayout(
        view, false, listOf("推荐", "世界"), listOf<Fragment>(BlogFragment(), BlogFragment()),
        savedInstanceState
    )
  }
}